samples/bluetooth/mesh/light_switch/overlay-lpn.conf

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_BT_MESH_LOW_POWER=y
+CONFIG_BT_DEVICE_NAME="Mesh Light Switch LPN"
+CONFIG_BT_MESH_RELAY=n
+CONFIG_BT_MESH_FRIEND=n
+
+# Power saving configuration:
+
+# Disable serial communication
+CONFIG_SERIAL=n
+CONFIG_UART_CONSOLE=n
+
+# Disable secure beacons
```

Comment:
Suggestion. This message basically says exactly what they next line does, but doesn't explain _why_ it is needed. Same for Poll Timeout and CONFIG_BT_MESH_GATT_PROXY_ENABLED. I think it will be more user friendly if the comment would explain a reason why default value is changed.